app/src/main/java/com/piwniczna/mojakancelaria/Models/RelationEntity.kt

```diff
@@ -2,16 +2,22 @@ package com.piwniczna.mojakancelaria.Models
 
 import androidx.room.ColumnInfo
 import androidx.room.Entity
+import androidx.room.ForeignKey
 import androidx.room.PrimaryKey
 import java.math.BigDecimal
 
 
-@Entity(tableName = "relations")
+@Entity(tableName = "relations", foreignKeys = [ForeignKey(entity = ClientEntity::class,
```

Comment:
obligations and payments id's are also foreign keys

Reply:
wixed